server/routers/activityRecord.ts

```diff
@@ -0,0 +1,85 @@
+import { TRPCError } from '@trpc/server';
+import { z } from 'zod';
+
+import { createTRPCRouter, privateProcedure } from '../trpc';
+
+export const activityRecordRouter = createTRPCRouter({
+  list: privateProcedure.query(async ({ ctx }) => {
+    const activityRecords = await ctx.prisma.activityRecord.findMany({
+      where: {
+        userId: ctx.user?.id,
+      },
+    });
+
+    return activityRecords;
+  }),
```

Comment:
If you have async function, that await for one thing and then just returns the result without any transformation/destructuring. It can be just returned right away, no waiting is needed, the "caller" will wait for it
```suggestion
  list: privateProcedure.query(({ ctx }) => ctx.prisma.activityRecord.findMany({
      where: {
        userId: ctx.user?.id,
      },
    })),
```

Reply:
I can see it in multiple places the same way. The question is if it does not hinder readability. The argument could also be that this is "slower/harder" when you want to add new functionality to the function - since you must add the explicit return again. But from experience, everybody prepares for future updates, which never come. And saving a few clicks is IMHO not worth it. 

But I will leave this up to you.

Reply:
this makes sense, but i guess that it will be changed for counting purposes later